.github/workflows/publish.yaml

```
@@ -0,0 +1,56 @@
+name: Publish
+on: push
+jobs:
+  version-changed:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
+          cache: 'yarn'
+      - name: cli
+        id: cli
+        uses: EndBug/version-check@v2
+        with:
+          diff-search: true
+          file-name: ./cli/package.json
+    outputs:
+      cli: ${{ steps.cli.outputs.changed }}
+  publish-cli:
+    runs-on: ubuntu-latest
+    needs: version-changed
+    if: ${{ needs.version-changed.outputs.cli }}
+    env:
+      # Only set 'latest' tag when pushing to main
+      npm_config_tag: ${{ github.ref == 'refs/heads/main' && 'latest' || 'experimental' }}
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
+          cache: 'yarn'
+      - name: Allow modern Yarn
+        run: |
+          corepack enable
+      - name: Install dependencies
+        run: |
+          yarn install
+      # Publish to NMP
+      - uses: actions/setup-node@v3
+        with:
+          registry-url: 'https://registry.npmjs.org'
+      - name: Publish to NPM
+        run: |
+          yarn workspace @iota-community/iota-wiki-cli npm publish --access public
```

And then do something like this here


```suggestion
          yarn workspace @iota-community/iota-wiki-cli npm publish --access public --tag $TAG --tolerate-republish $IS_MAIN
```

Would that work?

---

Why do we want the --tolerate-republish thingy?

---

Added with https://github.com/iota-wiki/iota-wiki/pull/785/commits/2b05a17d9ce5a17c7907128bb8fd796498e8bac5